Sources/FTPropertyWrappers/Keychain - Support/KeychainCoding.swift

```diff
@@ -0,0 +1,371 @@
+        case is Int64.Type:
+            return (try decode(from: data) as Int64) as! T
+        case is UInt.Type:
+            return (try decode(from: data) as UInt) as! T
+        case is UInt8.Type:
+            return (try decode(from: data) as UInt8) as! T
+        case is UInt16.Type:
+            return (try decode(from: data) as UInt16) as! T
+        case is UInt32.Type:
+            return (try decode(from: data) as UInt32) as! T
+        case is UInt64.Type:
+            return (try decode(from: data) as UInt64) as! T
+        case is Float.Type:
+            fatalError("Encoding root type Float not supported!")
+        case is Float80.Type:
+            fatalError("Encoding root type Float80 not supported!")
+        case is Double.Type:
+            fatalError("Encoding root type Double not supported!")
+        case is Bool.Type:
+            return (try decode(from: data) as Bool) as! T
+        case is String.Type:
+            return (try decode(from: data) as String) as! T
+        case is URL.Type:
+            return (try decode(from: data) as URL) as! T
+        case is Data.Type:
+            return (try decode(from: data) as Data) as! T
+        default:
+            return try decoder.decode(T.self, from: data)
+        }
+
+    }
+
+    // MARK: - Encoders overloaded
+    private func encode(_ value: Int) throws -> Data {
+        var data = Int64(value)
+        return Data(bytes: &data, count: MemoryLayout<Int64>.size)
+    }
+
+    private func encode(_ value: Int8) throws -> Data {
+        var data = Int64(value)
+        return Data(bytes: &data, count: MemoryLayout<Int64>.size)
+    }
+
+    private func encode(_ value: Int16) throws -> Data {
+        var data = Int64(value)
+        return Data(bytes: &data, count: MemoryLayout<Int64>.size)
+    }
+
+    private func encode(_ value: Int32) throws -> Data {
+        var data = Int64(value)
+        return Data(bytes: &data, count: MemoryLayout<Int64>.size)
+    }
+
+    private func encode(_ value: Int64) throws -> Data {
+        var data = value
+        return Data(bytes: &data, count: MemoryLayout<Int64>.size)
+    }
+
+    private func encode(_ value: UInt) throws -> Data {
+        var data = UInt64(value)
+        return Data(bytes: &data, count: MemoryLayout<UInt64>.size)
+    }
+
+    private func encode(_ value: UInt8) throws -> Data {
+        var data = UInt64(value)
+        return Data(bytes: &data, count: MemoryLayout<UInt64>.size)
+    }
+
+    private func encode(_ value: UInt16) throws -> Data {
+        var data = UInt64(value)
+        return Data(bytes: &data, count: MemoryLayout<UInt64>.size)
+    }
+
+    private func encode(_ value: UInt32) throws -> Data {
+        var data = UInt64(value)
+        return Data(bytes: &data, count: MemoryLayout<UInt64>.size)
+    }
+
+    private func encode(_ value: UInt64) throws -> Data {
+        var data = value
+        return Data(bytes: &data, count: MemoryLayout<UInt64>.size)
+    }
```

Comment:
I would remove throws keyword from all not throwing methods. The functions don't need to have the same signature.